### ISSUE TITLE
Submit updated Food Oasis project details to Hack for LA Website Team 

### ISSUE THREAD

**Issue by staceyrebekahscott** [hackforla/food-oasis#1269]
### Overview
Details about the Food Oasis project on the HfLA website needs to be updated to reflect current team, links and Overview details. 

### Action Items
- [x] Verify details in Overview
  - [x] Status
  - [x] Partners
  - [x] Location
  - [x] Tools
  - [x] Languages
  - [x] Technologies
  - [x] Meetings
  - [x] Project descriptions
- [x] Verify Resources are relevant and links work properly
- [x] Verify Project Current Team - check Team Roster to confirm current team members

### Resources/Instructions
[Food Oasis page on HfLA website](https://www.hackforla.org/projects/food-oasis)
Original Issue (closed as outdated) #489

**Comment by staceyrebekahscott** [comment]
Currently assigned to myself, but will pass along to Gordon Ruby once it is confirmed he is has been added to the Food Oasis repository.

**Comment by staceyrebekahscott** [comment]
@yiminng Would you be able to work on this issue as part of general PM project organization work? You will need to work with someone on the HfLA website team to follow the proper process. And it would be worth it to have the process documented in our Drive for future updates.

**Comment by staceyrebekahscott** [comment]
@yiminng Checking in to see if you would like to take this issue and work to update the HfLA website for Food Oasis. The Website team has an issue template for this, you'll have to reach out to find out the proper process to follow.

**Comment by yiminng** [comment]
sounds good!  I will contact website team @staceyrebekahscott

**Comment by staceyrebekahscott** [comment]
@yiminng Checking in- Will you be able to make progress on this issue this week?

**Comment by yiminng** [comment]
I've contacted the team on slack however no one has responded to it. I found an old issue https://github.com/hackforla/website/pull/3327, and commented here

**Comment by staceyrebekahscott** [comment]
@yiminng Note that you updated a Pull Request, not an Issue. 

If you go to the Website project's board, go to Issues -> New Issue -> there is an issue called: Project Profile Card review and update. Start a new issue on the Website project board for Food Oasis, and use the old issue # 3232 and follow the format for what needs to be updated. 

For who should be listed now: 

Stacey Rebekah Scott: Project/ Product Manager
Erin Dean: Product Manager
Yi Min Ng: Product Manager
Rylan Talerico: Product Manager
Gigi Patel: Lead UX Researcher
Seiko Igi: Lead UX Designer
Bryan Wu: UI/UX Subject Matter Expert
John Darragh: Lead Developer
Hannah Zulueta: Lead Developer

You'll have to search on Slack for the ID, as they ask for that info. 

Once the issue is updated, follow up again on Slack to notify the website team that you created the issue. Based on the previous issue, it took some time before the issue was completed, so make a note to stay on top of it to make sure it gets done.

**Comment by staceyrebekahscott** [comment]
@yiminng Also a reminder to check the other items listed in the issue Overview. We want to make sure to check the resource links and the overview details to make sure they are current. [Food Oasis on HfLA Website](https://www.hackforla.org/projects/food-oasis)

**Comment by staceyrebekahscott** [comment]
Sent note to Yi Min via Slack suggesting collaborating on this, as it might be easier to work together.

**Comment by staceyrebekahscott** [comment]
@yiminng Please make the following updates to the website issue (I cannot edit it): https://github.com/hackforla/website/issues/3572

Tools: Figma, whiteboard / paper & pencil, photoshop, phone calls, paper prototyping, Google forms, FigJam, Usabilityhub.com surveys, Excel, R, Google Analytics, SPSS, Mplus, AtlasTI

Languages: JavaScript, PL/pgSQL, SCSS, CSS, HTML, Dockerfile, Shell

Technologies: React, Express, Node.js, PostgreSQL, PostGIS, AWS, Docker, Mapbox, Material-UI

Update roles: 
Stacey Rebekah Scott
Senior Project Manager

Erin Dean
Senior Product Manager

**Comment by yiminng** [comment]
Information update on issue, messaged slack channel re: issue

**Comment by staceyrebekahscott** [comment]
@yiminng What is the status on this? If you're not having any luck on the Slack channel, maybe try searching through the In Progress issues on the Website team's Kanban and sending an @ mention to someone from Issue 3572 to alert them that the issue needs attention. Ask how long these updates typically take so we can have some expectations.

**Comment by yiminng** [comment]
messaged 2 people, one responded. Now in communication with @blulady it is now on "New Issue Approval" on the Website team github.

**Comment by staceyrebekahscott** [comment]
@itserindean Considering how time consuming following up on this issue has been, I am inclined to close this issue until the website team can come up with a more efficient way for us to implement these changes. If you concur with this, I will note and mark this issue as Done.

**Comment by itserindean** [comment]
What if we just moved it to icebox? It's in the hands of website team, correct? We're just waiting on them? I don't think marking it as done is appropriate since it's not completed.

**Comment by staceyrebekahscott** [comment]
I see this as completed on our end, as there is nothing else for us to provide for them and I don't like issues like this cluttering our backlog. But I will concede (for now) and put it in the Icebox. But if there is no progress in the next few months, I'll push for us to close it- if it is not a priority for the website team, it shouldn't be a priority for us.

**Comment by itserindean** [comment]
What if we changed the title to a better version of "submit info to ___ team with how we want website" Then I agree that we've completed this issue....

**Comment by staceyrebekahscott** [comment]
Great compromise. 

Updated issue title to "Submit updated Food Oasis project details to Hack for LA Website Team" 

Marking issue closed as completed.
